BatteryLevel.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Checker
+{
+    class BatteryLevel
+    {
+       public enum BatteryCondition
+       {
+           Low,
+           Normal,
+           High
+       }
+
+       public static BatteryCondition CheckBatteryCondition(float currentValue, float minValue, float maxValue)
```

Comment:
why don't you separate max check and min check in to different pure functions ?

Reply:
Thank you for your suggestion. The code has been separated for checking max and min check accordingly